content/posts/contribuindo_com_projetos_abertos.md

```diff
@@ -19,7 +19,112 @@ categories = [
 series = ["Tutoriais"]
 
 [ author ]
-  name = "Beatriz Val Franco"
+  name = "Tainara Costa"
 +++
 
-1. Coloque aqui
\ No newline at end of file
+Para essa receita você vai precisar de: acesso à internet, uma conta no GitHub, ter o Git instalado no seu computador ([saiba mais aqui](https://git-scm.com/book/pt-br/v2/Come%C3%A7ando-Instalando-o-Git)), criatividade para colaborar com projetos e uma pitada de curiosidade.
+   
+**_Tá a fim de compartilhar conhecimento? Então vem!_**
+
+![gif](https://media.giphy.com/media/NXp9HM6YeuS0U/giphy.gif)
```

Comment:
Em geral não é bom referenciar links de imagens de outros sites. Eles podem retirar a imagem do ar, ou modificar a URL. Se isso for algo recorrente, no futuro teremos centenas de referências quebradas e será um trabalhão corrigir tudo. O ideal é você baixar a imagem, nesse caso o `.gif` e salvá-lo dentro do projeto. Sempre é bom colocar a referência das imagens também, vou ver um jeito legal para fazermos isso.